src/types.ts

```diff
@@ -822,6 +822,10 @@ export type MuteUserOptions<UserType = UnknownType> = {
 };
 
 export type PaginationOptions = {
+  created_at_after?: string | Date | SeamlessImmutable.ImmutableDate | null;
+  created_at_after_or_equal?: string | Date | SeamlessImmutable.ImmutableDate | null;
+  created_at_before?: string | Date | SeamlessImmutable.ImmutableDate | null;
+  created_at_before_or_equal?: string | Date | SeamlessImmutable.ImmutableDate | null;
```

Comment:
we probbaly don't need the null type here
```suggestion
  created_at_after?: string | Date | SeamlessImmutable.ImmutableDate;
  created_at_after_or_equal?: string | Date | SeamlessImmutable.ImmutableDate;
  created_at_before?: string | Date | SeamlessImmutable.ImmutableDate;
  created_at_before_or_equal?: string | Date | SeamlessImmutable.ImmutableDate;
```